src/Propel/Generator/Reverse/PgsqlSchemaParser.php

```diff
@@ -309,7 +309,9 @@ protected function addColumns(Table $table, $oid)
             }
 
             if (null !== $default) {
-                if ("'" !== substr($default, 0, 1) && strpos($default, '(')) {
+                $containsFunctionCall = "'" !== substr($default, 0, 1) && strpos($default, '(');
+                $isCurrentTimestamp = $default === 'CURRENT_TIMESTAMP';
```

Comment:
There are a few experessions were introduced
current_date
current_time
current_timestamp
localtime
localtimestamp

Are they handled elsewhere?

Reply:
Are you sure you get the only uppercase?

Reply:
I think all of those should get added but I'd like to add tests to make sure they'll work over all supported platforms still.